style.css

```diff
@@ -0,0 +1,90 @@
+.dots {
+    width: 30px;
+    height: 40px;
+    position: relative;
+    left: 300px;
+    top: -170px;
+    background-image: radial-gradient(circle, black 4px, transparent 5px);
+    background-size: 100% 33.33%;
+}
+
+.texttile {
+    position: relative;
```

Comment:
Using `position: relative` on all your elements is a very brittle way of doing CSS layout because all it takes is for an element to be larger than anticipated or a line to have more text than in your design for things to get very much out of whack. For example, this is what happens to your current design if I add another sentence inside the "textile" `div`:

<img width="385" alt="Screen Shot 2020-11-07 at 12 43 00 PM" src="https://user-images.githubusercontent.com/2653980/98447950-dc008f80-20f6-11eb-9c14-1fdbe30714a2.png">

Instead of relying so heavily on relative positioning, you can use CSS properties like `display` and `vertical-align` to get the same result so that, if something changes in your content, things work with that, like so:

<img width="343" alt="Screenshot_2020-11-07 Project2-CardDesign" src="https://user-images.githubusercontent.com/2653980/98448077-a7d99e80-20f7-11eb-926e-18a8429b5665.png">

In [this video](https://www.loom.com/share/303d3e2ec6394b198fb0433d430f7bf3), I walk through a different way to approach your layout and here is the [Glitch project](https://glitch.com/~pocky-project2-layout) I remixed from your Project 2. Let me know if you have any questions about these changes.